app/cypress/integration/cif/project-revision-change-logs.spec.js

```diff
@@ -31,5 +31,15 @@ describe("the project amendment and revisions page", () => {
     cy.url().should("include", "orderBy=REVISION_TYPE_DESC"); //just to wait for the page to load
     cy.get("tbody tr").first().contains("Minor Revision");
     cy.get("tbody tr").last().contains("In Discussion");
+
+    // checking the view page for a specific revision
+    cy.findAllByRole("button", { name: /^view$/i })
+      .first()
+      .click();
+    cy.url().should("include", "/view");
+    cy.happoAndAxe("Amendment & Revision View", "view", "main", true);
```

Comment:
This screenshot returns an empty loading page, we should probably wait for content before taking it